Documentation/Storage-Configuration/Object-Storage-RGW/cosi.md

```diff
@@ -79,7 +55,7 @@ metadata:
 driverName: ceph.objectstorage.k8s.io
 deletionPolicy: Delete
 parameters:
-  objectStoreUserSecretName: rook-ceph-object-user-my-store-cosi
```

Comment:
I see the prefix is added back to the prefix in the `GenerateCephUserSecretName()` method, but needs to be updated in the examples in a bunch of places like this.